data_row_major: true
sensor:
  hfov: !degrees 87
  vfov: !degrees 58
  max_range: 5
box:
  min_x: -17
  max_x: 17
  min_y: -12
  max_y: 7
  min_z: 0.0
  max_z: 14.5